### DISCUSSION TITLE
Form in Paste

### DISCUSSION THREAD

**Discussion by uditadhikari117** [twilio-labs/paste#2621] [General] [answered]
Hi,
I am checking Paste first to see if Paste has a form. I have found pill form widget. But I found no other form widget that could support the signup form. Following is the screenshot of the signup form : 

![Screen Shot 2022-08-19 at 2 28 58 PM](https://user-images.githubusercontent.com/97563897/185684065-ecfda596-28dc-4f16-ad60-3ce3738f5a8b.png)

**Comment by loreina** [comment, marked as answer]
Hi @uditadhikari117,

We currently have a ticket in our backlog for guidance on form layouts. In the meantime, you can use Paste components to compose the form in the screenshot such as [Input](https://paste.twilio.design/components/input), [Checkbox](https://paste.twilio.design/components/checkbox), and [Button](https://paste.twilio.design/components/button).

I also created a mockup in codesandbox as an example of creating the form using Paste: https://codesandbox.io/s/nostalgic-leavitt-l5rc70?file=/src/index.tsx

Hope this helps!
